scalafmt-core/shared/src/main/scala/org/scalafmt/config/ReaderUtil.scala

```diff
@@ -7,16 +7,29 @@ import metaconfig.Configured._
 
 object ReaderUtil {
   // Poor mans coproduct reader
-  def oneOf[T: ClassTag](options: sourcecode.Text[T]*): ConfDecoder[T] = {
-    val m = options.map(x => x.source.toLowerCase() -> x.value).toMap
+  def oneOf[T: ClassTag](options: sourcecode.Text[T]*): ConfDecoder[T] =
+    oneOfImpl(lowerCase, options)
+
+  def oneOfIgnoreBackticks[T: ClassTag](
+      options: sourcecode.Text[T]*): ConfDecoder[T] =
+    oneOfImpl(lowerCaseNoBackticks, options)
+
+  private val lowerCase: String => String = s => s.toLowerCase
```

Comment:
Why not `def lowerCase(s: String): String = s.toLowercase`?

There are many reasons to use methods instead of functions. Function declarations are harder to read, produce more cryptic stack traces, `val` in objects are evaluated during global initialization potentially causing NullPointerException, `val` in object creates a JVM field + method while methods creates only one JVM method.